### PR TITLE
Adjust for new location of kubectl.sh and local-up-cluster.sh scripts in CI

### PR DESCRIPTION
**Pull request by dims**
Print where kubernetes-src.tar.gz gets laid down as well.

### CODE REVIEW

File: kubetest/local.go [hunk 1]
```diff
@@ -52,13 +53,32 @@ func newLocalCluster() *localCluster {
...
+		return "", err
+	}
+	path := filepath.Join(cwd, scriptPath)
+	if _, err := os.Stat(path); err == nil {
+		return path, nil
+	}
+	path = filepath.Join(cwd, "kubernetes", scriptPath)
+	if _, err := os.Stat(path); err == nil {
+		return path, nil
+	}
```
Comment (krzyzacy) [inline kubetest/local.go]: are those path experimental just to see where to get the script? I'd assume only one of the paths will work.

Comment (dims) [reply kubetest/local.go]: @krzyzacy - yes, we pick the first one that works and use that.

Comment (krzyzacy) [reply kubetest/local.go]: you probably can log down which path it picked, and clean up the unneeded one afterwards?

Comment (dims) [reply kubetest/local.go]: @krzyzacy So i am expecting the second one to work in the CI. The first one works in a normal dev box where you checkout k/k and run `make cross`. So we will end up needing both i am afraid. However, let's get this in and confirm the theory.

Comment (krzyzacy) [reply kubetest/local.go]: ok :-)

**Pull request merged**